Dockerfile_ubuntu

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:18.04
```

Comment:
Just a dumb question, but why do we want an ubuntu image? Can't we just use for now the alpine version?